scripts/broker-ci/postgresql.yaml

```diff
@@ -4,7 +4,7 @@ metadata:
   name: postgresql
   namespace: default
 spec:
-  serviceClassName: dh-ansibleplaybookbundle-rhscl-postgresql-apb
+  serviceClassName: dh-ansibleplaybookbundle-rhscl-postgresql-apb-lates
```

Comment:
It's not a typo :)

The name can at most be 51 characters if the catalog is not to complain. When it is created ithrough the normal process it will be `dh-ansibleplaybookbundle-rhscl-postgresql-apb-lates` not `dh-ansibleplaybookbundle-rhscl-postgresql-apb-latest`.

If this says latest it will actually fail.
```echo -ne "dh-ansibleplaybookbundle-rhscl-postgresql-apb-lates" | wc -c
51
```